src/commands/Games/wheeloffortune.ts

```diff
@@ -0,0 +1,47 @@
+import { MessageAttachment, MessageEmbed } from 'discord.js';
+import { CommandStore, KlasaMessage } from 'klasa';
+import { SkyraCommand } from '../../lib/structures/SkyraCommand';
+import { UserSettings } from '../../lib/types/settings/UserSettings';
+import { Emojis } from '../../lib/util/constants';
+import { WheelOfFortune } from '../../lib/util/Games/WheelOfFortune';
+import { getColor } from '../../lib/util/util';
+
+export default class extends SkyraCommand {
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			aliases: ['wof'],
+			cooldown: 10,
+			description: language => language.tget('COMMAND_WHEELOFFORTUNE_DESCRIPTION'),
+			extendedHelp: language => language.tget('COMMAND_WHEELOFFORTUNE_EXTENDED'),
+			requiredPermissions: ['EMBED_LINKS'],
```

Comment:
You need to add `'EMBED_LINKS'` to slotmachine, and `'ATTACH_FILES'` in this one.